src/components/views/articles.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <div>
+        <div class="galery">
+            <OrganismArticle
+                v-for="i in 9"
+                src="https://image.geo.de/30147338/t/iL/v4/w1440/r0/-/nilpferd-wildlife-photographer-of-the-year-jpg--83937-.jpg"
+                name="Banane"
+                amount="7"
+                expiringIn="10"
+                mhd="29.02.2022"
+                storage="Kühlschrank"
+            ></OrganismArticle>
+        </div>
+    </div>
+</template>
+
+<script lang="ts" setup>
+import OrganismArticle from '../organisms/OrganismArticle.vue';
+</script>
+
+<style>
+.galery {
```

Comment:
Spelling change to "gallery"
```suggestion
.gallery {
```